CMakeLists.txt

```diff
@@ -127,6 +127,15 @@ IF(NOT WIN32)
   ELSE()
     MESSAGE(FATAL_ERROR "Driver Manager was not found")
   ENDIF()
+
+  # Looking for iconv files
+  INCLUDE(${CMAKE_SOURCE_DIR}/cmake/FindIconv.cmake)
```

Comment:
Shouldn't we have here something like IF(APPLE), since (at the moment) we don't need that dependency on other platforms?